pkg/plugins/platform/router.go

```diff
@@ -0,0 +1,92 @@
+package platform
```

Comment:
I agree with the spirit of what you are saying but I'm not sure I agree on the specifics. I think in general we could benefit from thinking through our packages a lot more than we actually have. I also think that the "platform as a catchall" system we're currently using is a bad strategy in the long term, and I'd love to spend some time reorganizing things a bit.

In particular, I'd love to remove the word "app" from our package vocabulary. It just ends up being a filler word that has no meaning. We also have an inversion problem where our "platform" loads our "config" struct. Ideally, we'd load the config first, and according to config values we load the appropriate platform. Finally, what this touches on, we end up with a lot of general site-util stuff like what is pointed out here in our platform package. I agree that it doesn't quite feel right here, but currently the `ddevapp` package contains nothing but config related stuff, so I don't think moving it there works either.

As a long term strategy, I think sitting down and thinking through the relationship between the current `ddevapp`, `appimport`, `appports`, and `platform` packages needs to be done eventually.